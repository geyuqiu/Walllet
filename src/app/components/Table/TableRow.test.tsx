import {render} from "@testing-library/react";
import React from "react";

import {TableRow} from "./TableRow";
import {TableCell} from './TableCell';

describe("TableRow", () => {
	it("should render with border", () => {
		const { container } = render(
			<table>
				<tbody>
					<TableRow border={true}>
						<TableCell>
							something
						</TableCell>
					</TableRow>
				</tbody>
			</table>,
		);
		expect(container).toMatchSnapshot();
	});

	it("should render without border", () => {
		const { container } = render(
			<table>
				<tbody>
					<TableRow border={false} >
						<TableCell>
							something
						</TableCell>
					</TableRow>
				</tbody>
			</table>,
		);
		expect(container).toMatchSnapshot();
	});
});
