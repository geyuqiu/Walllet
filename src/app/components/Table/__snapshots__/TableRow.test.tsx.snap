// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableRow should render with border 1`] = `
<div>
  <table>
    <tbody>
      <tr
        class="TableRow__TableRowStyled-sc-1p89o19-0 iRdKeT"
        data-testid="TableRow"
      >
        <td>
          <div
            class="TableCell__TableCellInnerWrapper-sc-12y9mmp-0 dbZfon"
          >
            something
          </div>
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;

exports[`TableRow should render without border 1`] = `
<div>
  <table>
    <tbody>
      <tr
        class="TableRow__TableRowStyled-sc-1p89o19-0 dXhSPI"
        data-testid="TableRow"
      >
        <td>
          <div
            class="TableCell__TableCellInnerWrapper-sc-12y9mmp-0 dbZfon"
          >
            something
          </div>
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
